eagle-core/eagle-alert-parent/eagle-alert/alert-engine/src/test/java/org/apache/eagle/alert/engine/publisher/AlertFilePublisherTest.java

```diff
@@ -0,0 +1,44 @@
+package org.apache.eagle.alert.engine.publisher;
+
+import org.apache.eagle.alert.engine.coordinator.Publishment;
+import org.apache.eagle.alert.engine.model.AlertStreamEvent;
+import org.apache.eagle.alert.engine.publisher.impl.AlertFilePublisher;
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+public class AlertFilePublisherTest {
+    private static final String TEST_POLICY_ID = "testPolicy";
+
+    @Test
+    public void testAlertFilePublisher() throws Exception {
+        Map<String, Object> properties = new HashMap<>();
+        properties.put(PublishConstants.ROTATE_EVERY_KB, 1);
+        properties.put(PublishConstants.NUMBER_OF_FILES, 1);
+
+        String property = "java.io.tmpdir";
+        String tempDir = System.getProperty(property);
+        System.out.println("OS current temporary directory is " + tempDir);
+
+        properties.put(PublishConstants.FILE_NAME, tempDir+"eagle-alert.log");
```

Comment:
Seems we need a system property "file.separator" between tempDir and "eagle-alert.log". Please check it.